docs/query-getfacilitieslist.html

```
@@ -186,25 +495,25 @@ <h1>GetFacilitiesList</h1><p class="text-muted">Get Facilities List</p>
 <div class="row">
     <div class="col-md-8">
         <div class="mb-3"><strong>Produced By:</strong> <a href="service-tenantservice.html">Tenant Service [service]</a>, <a href="service-adminbff.html">Admin Backend-for-Frontend [service]</a></div><div class="mb-3"><strong>Consumed By:</strong> <a href="service-tenantservice.html">Tenant Service [service]</a></div>
-        <h2 id="overview">Overview <a id="overview" href="#overview" class="markdown-heading-anchor text-decoration-none" aria-label="Link to section: Overview"><i class="bi bi-link-45deg" style="opacity: 0.2; font-size: 20px;"></i></a></h2>
+        <h2 id="overview" class="content-heading">Overview <a id="overview" href="#overview" class="markdown-heading-anchor text-decoration-none" aria-label="Link to section: Overview"><i class="bi bi-link-45deg" style="opacity: 0.2; font-size: 20px;"></i></a></h2>
```

HTMLHint (1.9.2)

[error] The id value [ overview ] must be unique.

(id-unique)